manual/mapper/daos/delete/README.md

```diff
@@ -105,6 +105,15 @@ The method can return:
     ResultSet deleteIfDescriptionMatches(UUID productId, String expectedDescription);
     // if the condition fails, the result set will contain columns '[applied]' and 'description'
     ```
+  
+* a [BoundStatement]. This is intended for queries with custom IF clauses; where you will execute
+  this statement later or in a batch.
+  
+    ```java
+    @Delete(entityClass = Product.class, customIfClause = "description = :expectedDescription")
+    BoundStatement deleteIfDescriptionMatches(UUID productId, String expectedDescription);
+    // if the condition fails, the result set will contain columns '[applied]' and 'description'
+    ```
```

Comment:
> This is intended for queries with custom IF clauses

I don't see why, is this a copy-paste error?

If so we should also modify the example to something simpler. A simple `BoundStatement delete(Product product)` should suffice.

Reply:
That makes sense, since the primary goal of this feature it to give you the bound statement to do what you will with it later. It's just a bonus that you can also use conditionals etc too if you choose.